os/os.proto

```diff
@@ -7,7 +7,7 @@ import "types/types.proto";
 
 option go_package = "github.com/openconfig/gnoi/os";
 
-option (types.gnoi_version) = "0.1.2";
+option (types.gnoi_version) = "0.1.3";
```

Comment:
lets bump it to 4 given that there is another push that will get in before this one and is already claiming 3.